iphone/TitaniumKit/TitaniumKit/Sources/API/TiModule.m

```diff
@@ -21,6 +21,9 @@ - (void)unregisterForNotifications
 
 - (void)dealloc
 {
+  // Allow JavascriptCore to release module proxy.
+  [self forgetSelf];
```

Comment:
Since all native modules will be cached until the KrollBridge shuts down anyway it should be ok to remove the above lines and just stick with the `rememberSelf`.

Reply:
I'd like to keep the `forgetSelf` in `dealloc` since it could be possible to call `dealloc` on the KrollObject and there's no harm in keeping it it

Reply:
You never send a `dealloc` manually, it will be done automatically though retain/release. And since the JS object retains the KrollObject there is no way that it will be called in a way that makes sense.

Sure there is no harm in keeping it, but i would prefer not having code around that has no effect ;)